tensorflow/core/kernels/conv_ops_direct_sycl.h

```diff
@@ -0,0 +1,224 @@
+#ifndef TENSORFLOW_USE_SYCL
+#error This file should only be included when compiling with SYCL support
+#endif
+
+#ifndef TENSORFLOW_KERNELS_CONV_OPS_NAIVE_SYCL_H_
+#define TENSORFLOW_KERNELS_CONV_OPS_NAIVE_SYCL_H_
+
+#include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
+
+#include "tensorflow/core/kernels/conv_ops_sycl_common.h"
+
+#include "tensorflow/core/kernels/conv_ops_direct_sycl_kernels.h"
+#include "tensorflow/core/kernels/conv_ops_direct_sycl_nchw_kernels.h"
+
+namespace tensorflow {
+typedef Eigen::SyclDevice SYCLDevice;
+namespace direct {
+template <ConvType CType>
+inline SYCLConv2DParams get_kernel_params(SYCLConv2DParams params) {
```

Comment:
This function does absolutely nothing, is it declared for API stability with non sycl implementation?